src/community/features-templating/features-templating-ogcapi/src/test/java/org/geoserver/featurestemplating/response/JSONLDGetComplexFeaturesResponseAPITest.java

```diff
@@ -82,14 +83,14 @@ public void testJsonLdQueryEnvSubstitutionOnSource() throws Exception {
                 new StringBuilder("ogc/features/collections/")
                         .append("gsml:MappedFeature")
                         .append("/items?f=application%2Fld%2Bjson")
-                        .append("&env=source:NotMappedFeature");
```

Comment:
Changed in the template the level for which env parametrization is used in source.
Since the top level feature source is never validate against the featureType as a propertyName because will always, the test was actually testing for a fragility of the validation mechanism. now the tested source is one for a nested feature